core/lib/GlobalPool.ts

```diff
@@ -37,9 +32,6 @@ export default class GlobalPool {
   }[] = [];
 
   public static async start() {
-    if (this.isStarted) {
-      throw new Error('GlobalPool is already started');
-    }
```

Comment:
Why get rid of `isStarted`?

Reply:
We weren't using it anymore. GlobalPool no longer had a single startup requirement in the code. I don't know when we switched to booting based on if a chromeCore instance existed or not, but that's what the code does. I think this is legacy from Ulixee. 

I was tracking down an issue in Windows where it was still shutting down during the next startup, and cleared this out when I saw it didn't actually do anything anymore. We never set isStarted to true.

Reply:
Ok, yeah... crazy that it was doing nothing.